vmdb/app/models/vm_or_template.rb

```diff
@@ -1064,7 +1064,7 @@ def storage2active_proxies(all_proxy_list = nil)
     # MiqServer coresident proxy needs to contact the host and provide credentials.
     # Remove any MiqServer instances if we do not have credentials
     rsc = self.scan_via_ems? ? self.ext_management_system : self.host
-    proxies.delete_if {|p| MiqServer === p} if rsc && rsc.authentication_invalid?
+    proxies.delete_if {|p| MiqServer === p} if rsc && rsc.missing_credentials?
```

Comment:
Ok, so the concept of embedded proxy still stands...I thought that was thrown away too (or at least the concept that it's a proxy as opposed to just being the code that is called)